references/classification/train.py

```diff
@@ -30,23 +30,20 @@ def train_one_epoch(model, criterion, optimizer, data_loader, device, epoch, arg
     for i, (image, target) in enumerate(metric_logger.log_every(data_loader, args.print_freq, header)):
         start_time = time.time()
         image, target = image.to(device), target.to(device)
-        output = model(image)
+        with torch.cuda.amp.autocast(enabled=args.amp):
+            output = model(image)
+            loss = criterion(output, target)
```

Comment:
Good idea. Making both the forward and the loss estimation to be within the `autocast` call makes sense. [See documentation](https://pytorch.org/docs/stable/notes/amp_examples.html#typical-mixed-precision-training).

Reply:
Yeah, I agree. I think I missed it.